config/test.json

```diff
@@ -84,6 +84,7 @@
 		"reader/full-errors": true,
 		"reader/list-management": false,
 		"reader/new-post-notifications": true,
+		"reader/user-mention-suggestions": false,
```

Comment:
Probably fine to enable for testing, should you want to add component tests.

Reply:
Good shout 👍